static/src/javascripts/projects/common/modules/experiments/tests/commercial-us-mobile-sticky.js

```diff
@@ -0,0 +1,31 @@
+// @flow
+import { getCookie } from 'lib/cookies';
+import { isBreakpoint } from 'lib/detect';
+
+export const commercialUsMobileSticky: ABTest = {
+    id: 'CommercialUsMobileSticky',
+    start: '2019-06-23',
```

Comment:
You may want to set the start this week if you plan to demo it or opt-in to the AB test and see this on PROD

Pretty sure that the canRun checks on AB tests adhere to the dates 😸

Reply:
That was supposed to be 2019-05-23. Good catch. I'd be going mental trying to find what was wrong with it. Worked fine on CODE though.